next-sitemap.js

```diff
@@ -31,6 +27,7 @@ module.exports = {
       'https://next.egghead.io/tags-sitemap-4.xml',
       'https://next.egghead.io/tags-sitemap-5.xml',
       'https://next.egghead.io/tags-sitemap-6.xml',
+      'http://egghead-sitemaps.s3.amazonaws.com/sitemaps/sitemap.xml.gz',
```

Comment:
egghead-rails will still generate the content sitemap, we should make sure it generates it with the appropriate URL

Reply:
https://github.com/eggheadio/egghead-rails/blob/8419d61081cc5305be69285c165213ddd0a87174/config%2Fsitemap.rb

verified that the root url is configured in the sitemap config

Reply:
plugin link for convenience: https://www.npmjs.com/package/next-sitemap#full-configuration-example